bundle/deploy/terraform/init.go

```diff
@@ -219,8 +220,12 @@ func setProxyEnvVars(ctx context.Context, environ map[string]string, b *bundle.B
 }
 
 func setUserAgentExtraEnvVar(environ map[string]string, b *bundle.Bundle) error {
-	var products []string
+	// Add databricks-dabs to the user agent with the version of the CLI. This
+	// allows us to attribute HTTP request made by the Databricks Terraform provider
+	// to DABs.
+	dabsVersion := fmt.Sprintf("databricks-dabs/%s", build.GetInfo().Version)
 
+	products := []string{dabsVersion}
 	if experimental := b.Config.Experimental; experimental != nil {
 		if experimental.PyDABs.Enabled {
 			products = append(products, "databricks-pydabs/0.0.0")
```

Comment:
FYI, this name is equivalent to the PyPI package.

Reply:
Yeah, that makes sense. I assumed the implied convention here was databricks-{product-name}